cmd/globals.go

```diff
@@ -265,6 +265,7 @@ func getGlobalInfo() (globalInfo map[string]interface{}) {
 	globalInfo = map[string]interface{}{
 		"isWorm":       globalWORMEnabled,
 		"serverRegion": globalServerRegion,
+		"domains":      globalDomainNames,
```

Comment:
We may remove this globalInfo entirely, let's not extend its scope for now.

Reply:
👌